code/game/objects/items/plantable_flags.dm

```diff
@@ -1,60 +1,225 @@
+	return ..()
+
+/obj/item/plantable_flag/deconstruct(disassembled = TRUE, mob/living/blame_mob)
+	on_destroy(blame_mob)
+	return ..()
 
-/obj/item/flag_base/examine(mob/user)
+/obj/item/plantable_flag/toggle_deployment_flag(deployed)
 	. = ..()
-	. += "It has a flag made for the [country_name] inside it."
+	update_aura()
 
+/obj/item/plantable_flag/Moved()
+	. = ..()
+	update_aura()
+
+/obj/item/plantable_flag/attack_self(mob/user)
+	. = ..()
+	lift_flag(user)
+
+/obj/item/plantable_flag/ex_act(severity)
+	switch(severity)
+		if(EXPLODE_DEVASTATE)
+			take_damage(500, BRUTE, BOMB)
+		if(EXPLODE_HEAVY)
+			take_damage(150, BRUTE, BOMB)
+		if(EXPLODE_LIGHT)
+			take_damage(75, BRUTE, BOMB)
+		if(EXPLODE_WEAK)
+			take_damage(15, BRUTE, BOMB)
+
+/obj/item/plantable_flag/fire_act(burn_level)
+	take_damage(burn_level * 3, BURN, FIRE)
 
-/obj/item/flag_base/attack_self(mob/user)
-	to_chat(user, "<span class='warning'>You start to deploy the flag between your feet...")
-	if(!do_after(usr, 1 SECONDS, NONE, src, BUSY_ICON_BUILD))
-		to_chat(user, "<span class='warning'>You decide against deploying the flag here.")
+///Updates the aura strength based on where its currently located
+/obj/item/plantable_flag/proc/update_aura()
+	if(!current_aura)
 		return
+	current_aura.range = item_flags & IS_DEPLOYED ? FLAG_AURA_DEPLOYED_RANGE : FLAG_AURA_RANGE
+	if(isturf(loc))
+		current_aura.strength = LOST_FLAG_AURA_STRENGTH
+		return
+	if(!isliving(loc))
+		return
+	var/mob/living/living_holder = loc
+	if(living_holder.faction == faction)
+		current_aura.strength = FLAG_AURA_STRENGTH
+	else
+		current_aura.strength = LOST_FLAG_AURA_STRENGTH //this explicitly lets enemies deploy it for the extended debuff range
 
-	playsound(loc, 'sound/effects/thud.ogg', 100)
-	user.dropItemToGround(src)
-	is_collapsed = FALSE
-	update_appearance()
+///Sets debuff and assigns blame if the flag is destroyed
+/obj/item/plantable_flag/proc/on_destroy(mob/living/blame_mob)
+	SSaura.add_emitter(get_turf(src), AURA_HUMAN_FLAG, INFINITY, LOST_FLAG_AURA_STRENGTH, 900, faction)
 
+	if(istype(blame_mob) && blame_mob.ckey)
+		var/datum/personal_statistics/personal_statistics = GLOB.personal_statistics_list[blame_mob.ckey]
+		if(faction == blame_mob.faction) //prepare for court martial
+			personal_statistics.flags_destroyed --
+			personal_statistics.mission_flags_destroyed --
+		else
+			personal_statistics.flags_destroyed ++
+			personal_statistics.mission_flags_destroyed ++
 
-/obj/item/flag_base/attack_hand(mob/living/user)
-	if(!is_collapsed)
-		if(!do_after(usr, 1 SECONDS, NONE, src, BUSY_ICON_BUILD))
-			to_chat(user, "<span class='warning'>You decide against removing the flag here.")
-			return
-		is_collapsed = TRUE
-		update_appearance()
-	return ..()
+///Waves the flag around heroically
+/obj/item/plantable_flag/proc/lift_flag(mob/user)
+	if(TIMER_COOLDOWN_CHECK(user, COOLDOWN_WHISTLE_WARCRY))
+		to_chat(user, span_notice("You have to wait a while to rally your troops..."))
```

Comment:
Balloooon

Reply:
nou. We don't need balloon alerts for everything, this is just an antispam message

Reply:
Exactly, it is something you only need to know in the immediate moment

Reply:
balloon alert is way more stinky in this case but its changed.